models/reviews.models.js

```diff
@@ -27,3 +27,36 @@ exports.selectReviewById = review_id => {
       }
     });
 };
+
+exports.updateReviewById = (review_id, inc_votes) => {
+  if (inc_votes === '') {
```

Comment:
maybe worth checking if type of inc_votes value is a string not just strictly equal to an empty string? though this should be handled by SQL so if it's purely for purpose of descriptive errors then that's fine